modules/apps/headless/headless-batch-engine/headless-batch-engine-test/src/testIntegration/java/com/liferay/headless/batch/engine/resource/v1_0/test/ExportTaskResourceTest.java

```diff
@@ -656,6 +663,7 @@ else if (Objects.equals(
 		"com.liferay.object.admin.rest.dto.v1_0.ObjectRelationship",
 		"com.liferay.object.admin.rest.dto.v1_0.ObjectValidationRule",
 		"com.liferay.object.admin.rest.dto.v1_0.ObjectView",
+		"com.liferay.portal.language.rest.dto.v1_0.Message",
```

Comment:
I am not 100% sure that it should be added. It gets an error but no error message

Reply:
Let's disable it for now. We will be soon working on making this test better.